internal/suggest/diagnose.go

```diff
@@ -2,14 +2,12 @@ package suggest
 
 import (
 	"context"
-	"fmt"
 	"github.com/speakeasy-api/speakeasy-core/openapi"
 	"github.com/speakeasy-api/speakeasy-core/suggestions"
 	"github.com/speakeasy-api/speakeasy/internal/schema"
-	"strings"
 )
 
-func Diagnose(ctx context.Context, schemaPath string) (*suggestions.Diagnosis, error) {
+func Diagnose(ctx context.Context, schemaPath string) (suggestions.Diagnosis, error) {
```

Comment:
make it a pointer ?

Reply:
Doesn't need to be a pointer because it's a map